examples/automatic-labelling-from-repository/main.tf

```diff
@@ -0,0 +1,131 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+terraform {
+  required_version = "~> 0.11.0"
+}
+
+provider "archive" {
+  version = "~> 1.0"
+}
+
+provider "google" {
+  version = "~> 2.1"
+}
+
+provider "random" {
+  version = "~> 2.0"
+}
+
+provider "null" {
+  version = "~> 1.0"
+}
+
+resource "random_pet" "main" {
+  separator = "-"
+}
+
+resource "google_sourcerepo_repository" "main" {
+  name    = "${random_pet.main.id}"
+  project = "${var.project_id}"
+}
+
+resource "null_resource" "configure_repository" {
+  triggers {
+    repository_url = "${google_sourcerepo_repository.main.url}"
+  }
+
+  provisioner "local-exec" {
+    command = "${path.module}/scripts/configure_repository.sh"
+
+    environment {
+      REMOTE_URL                = "${google_sourcerepo_repository.main.url}"
+      REPOSITORY_DIRECTORY      = "${path.module}/function_source"
+      REPOSITORY_COPY_DIRECTORY = "${path.module}/function_source_copy"
+    }
+  }
+}
+
+data "null_data_source" "main" {
+  inputs = {
+    source_repository_url = "https://source.developers.google.com/projects/${var.project_id}/repos/${random_pet.main.id}/moveable-aliases/master/paths/"
+  }
+
+  depends_on = ["null_resource.configure_repository"]
+}
+
+module "event_project_log_entry" {
+  source = "../../modules/event-project-log-entry"
+
+  filter = "${
+    join(
+      " AND ",
```

Comment:
My work on the Forseti real time enforcer module indicates that we don't need explicit `AND` operators; they should be implied. It might be possible to drop the `join` call for the sake of simplicity.